code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -306,21 +306,21 @@
 		/obj/item/pipe_dispenser,
 		/obj/item/extinguisher,
 		/obj/item/weldingtool/largetank/cyborg,
-		/obj/item/screwdriver/cyborg,
-		/obj/item/wrench/cyborg,
-		/obj/item/crowbar/cyborg,
-		/obj/item/wirecutters/cyborg,
+		/obj/item/screwdriver/cyborg/power, // Skyrat Removal/Edit - Combines Screwdriver and Wrench into one
+		/obj/item/crowbar/cyborg/power, // Skyrat Removal/Edit - Combines Crowbar and Wirecutters into one
```

Comment:
Instead of this, please do it like this pseudocode:
make a new null list var in object/item/robot_model and name it skyrat_modules
for each model you add our custom items to the new list
on the initialize of the robot model you add the skyrat_modules to the basic_modules
profit

Reply:
Okey.

Reply:
In the case of the power tools, it's honestly the one moment I would say it would make sense, because it's replacing the other ones with the power tools versions, just to remove a bit of inventory clutter.

Reply:
Can I put the module sheets as a separate file in modular_skyrat?

Reply:
yes